integrations/github/src/haystack_integrations/components/connectors/github/pr_creator.py

```diff
@@ -0,0 +1,270 @@
+import re
+from typing import Any, Dict, Optional
+
+import requests
+from haystack import component, default_from_dict, default_to_dict, logging
+from haystack.utils import Secret, deserialize_secrets_inplace
+
+logger = logging.getLogger(__name__)
+
+
+@component
+class GithubPRCreator:
```

Comment:
I believe github is usually capitalized as GitHub, so worth considering updating our names I think
```suggestion
class GitHubPRCreator:
```

Reply:
Done.